airbyte-cdk/bulk/toolkits/load-s3/src/main/kotlin/io/airbyte/cdk/load/command/s3/S3PathSpecification.kt

```diff
@@ -50,27 +50,31 @@ interface S3PathSpecification {
     @get:JsonSchemaInject(json = """{"examples":["data_sync/test"]}""")
     val s3BucketPath: String
 
-    @get:JsonSchemaTitle("Use a Staging Directory")
-    @get:JsonPropertyDescription(
-        "Whether to use a staging directory in the bucket based on the s3_staging_prefix. If this is not set, airbyte will maintain sync integrity by adding metadata to each object."
-    )
-    @get:JsonProperty("use_staging_directory", defaultValue = "false")
-    val useStagingDirectory: Boolean?
+    //    Uncomment to re-enable staging
 
-    @get:JsonSchemaTitle("S3 Staging Prefix")
-    @get:JsonPropertyDescription(
-        "Path to use when staging data in the bucket directory. Airbyte will stage data here during sync and/or write small manifest/recovery files."
-    )
-    @get:JsonProperty("s3_staging_prefix")
-    @get:JsonSchemaInject(json = """{"examples":["__staging/data_sync/test"]}""")
-    val s3StagingPrefix: String?
+    //    @get:JsonSchemaTitle("Use a Staging Directory")
```

Comment:
Should we delete the commented code?

Here and bellow?

Reply:
We are planning to re-enable this once the release is stable, I'd rather leave the commented staging code clearly labeled.

(The issue is that rolling back from a connector with staging enabled, then running a sync, then rolling forward, can break it.)